src/styles/variables.scss

```
@@ -0,0 +1,4 @@
+$main-accent: #060b35;
+$secondary: #616070;
```

```suggestion
$secondary-accent: #616070;
```
$secondary what?) As you named all with "-accent" why did you skip it here?